training/training_code/train-max-model.sh

```diff
@@ -26,7 +26,9 @@ echo "Training data is stored in $DATA_DIR"
 # The WML stores work files in the $RESULT_DIR.
 echo "Training work files and results will be stored in $RESULT_DIR"
 
-echo "Installing prerequisite packages ..."
+echo "Installing prerequisite packages - 1..."
+pip install --user --no-deps -r training_prerequirements.txt
+echo "Installing prerequisite packages - 2..."
```

Comment:
Why is this necessary? Could you please add a comment explaining

Reply:
Cython package is required for pycocotools installation. Some updates have been made to WML because of which Cython package is no longer available by default. Chosen this way to resolve the error

Reply:
Cython can't just be the first line of the file? It needs to be 2 files?

Reply:
Cython is the first package in the current version.  

Error when having all packages in one requirements file

```
Installing the following prerequisite packages ...
Cython==0.29.14
pycocotools==2.0.0

Collecting Cython==0.29.14
  Downloading Cython-0.29.14-cp36-cp36m-manylinux1_x86_64.whl (2.1 MB)
Collecting pycocotools==2.0.0
  Downloading pycocotools-2.0.0.tar.gz (1.5 MB)
    ERROR: Command errored out with exit status 1:
     command: /opt/anaconda/envs/wmlce/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-xi3xmmdo/pycocotools/setup.py'"'"'; __file__='"'"'/tmp/pip-install-xi3xmmdo/pycocotools/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-install-xi3xmmdo/pycocotools/pip-egg-info
         cwd: /tmp/pip-install-xi3xmmdo/pycocotools/
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-xi3xmmdo/pycocotools/setup.py", line 2, in <module>
        from Cython.Build import cythonize
    ModuleNotFoundError: No module named 'Cython'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

Pip starts with pycocotools installation
Splitting the requirements file is one of the options to resolve this error.